charts/quickwit/values.yaml

```diff
@@ -326,53 +371,84 @@ bootstrap:
 
   # Extra env for boostrap jobs
   extraEnv: {}
-    # KEY: VALUE
+  # KEY: VALUE
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed